src/components/DetailsCard/DetailsItem.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { lazy, Suspense } from 'react';
+import { Box, Typography, makeStyles } from '@material-ui/core/';
+import { FC } from '../../typings/components';
+
+const useStyles = makeStyles((theme) => ({
+  wrapper: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  label: {
+    marginLeft: theme.spacing(1),
+  },
+  icon: {
+    height: 16,
+    width: 16,
+  },
+}));
+
+type Props = { label: string; icon: string; className?: string; isLink?: boolean; href?: string };
+
+export const DetailsItem: FC<Props> = ({ label, icon, className = '', isLink = false, href }) => {
+  const classes = useStyles();
+  const renderIcon = () => {
+    const SelectedIcon = lazy(() => import(`react-feather/dist/icons/${icon}.js`));
```

Comment:
As we're using only 5 icons, importing each of them dynamically could be a little overengineering

Reply:
Do you think that I should change it or leave as it is? 😄

Reply:
I think you can change it, it will result in loading a 1KB of icon chunk with 5KB of webpack code, so it wouldn't be optimized well

Reply:
Changed ✅